plugins/pdr_deterministic_plugin/ufm_sim_web_service/isolation_mgr.py

```diff
@@ -815,18 +822,18 @@ def main_flow(self):
                     for port_state in list(self.ports_states.values()):
                         state = port_state.get_state()
                         cause = port_state.get_cause()
-                        # EZ: it is a state that say that some maintenance was done to the link 
+                        # EZ: it is a state that say that some maintenance was done to the link
                         #     so need to re-evaluate if to return it to service
                         if self.automatic_deisolate or cause == Constants.ISSUE_OONOC or state == Constants.STATE_TREATED:
                             self.eval_deisolate(port_state.name)
                 ports_updated = self.update_ports_data()
                 if ports_updated:
                     self.update_telemetry_session()
                 t_end = time.time()
-            except Exception as e:
+            except Exception as exception: #pylint: disable=broad-except
```

Comment:
Please put the disable=XXX in the line prior to the error because it is less visible at the end of the line.